vector/src/main/java/im/vector/app/features/home/room/detail/TimelineViewModel.kt

```diff
@@ -1023,6 +1038,47 @@ class TimelineViewModel @AssistedInject constructor(
         _viewEvents.post(RoomDetailViewEvents.NavigateToEvent(targetEventId))
     }
 
+    private fun handlePinMessage(action: RoomDetailAction.PinMessage) {
+        if (room == null) return
+        val idsOfPinnedMessages = getIdsOfPinnedEvents()
+        if (idsOfPinnedMessages == null) return
```

Comment:
This will be moved to the SDK, but in the case of `idsOfPinnedMessages` is null, should we just create the State event?